openapi.yaml

```diff
@@ -177,7 +178,7 @@ components:
       type: object
       properties:
         indexId:
-          description: Unique identifier for the constituent used to calculate the index.
+          description: Unique identifier for the resource.
```

Comment:
`Unique identifier for the constituent used to calculate the index.`. Is correct for GWA and MWA indices, we could add `Unique identifier for the constituent used to calculate the index. Or a unique ID for the index in the case of LX indices.`?

Reply:
Done